src/material/core/style/_vendor-prefixes.scss

```diff
@@ -43,4 +43,14 @@
   position: -webkit-sticky #{if($important, '!important', '')};
   position: sticky #{if($important, '!important', '')};
 }
+
+@mixin color-adjust($value) {
+  -webkit-print-color-adjust: $value;
+  color-adjust: $value;
+}
+
+@mixin background-clip($value) {
+  -webkit-background-clip: $value;
+  background-clip: $value;
+}
```

Comment:
Can we mark these as private for the time being? Until we get Google fully switched over to the new Sass API, we can't stop people from importing these